### PR TITLE
[CMS-584] Update badge contents and add pr-body-unsupported option.

### PR DESCRIPTION
**Pull request by kporras07**
### Overview
This pull request updates support level badges content and adds a new option to allow a different pr-body if support level is unsupported.

### CODE REVIEW

**Comment by kporras07** [comment]
Created as draft waiting for docs to approve the badges docs page.

File: src/Util/SupportLevel.php [hunk 1]
```diff
@@ -37,13 +37,14 @@ public static function getSupportLevelBadge($level)
...
-            'minimal' => '[![Minimal Support](https://img.shields.io/badge/pantheon-minimal_support-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/minimal-support?q=org%3Apantheon-systems)',
-            'unsupported' => '[![Unsupported](https://img.shields.io/badge/pantheon-unsupported-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/unsupported?q=org%3Apantheon-systems)',
-            'unofficial' => '[![Unofficial](https://img.shields.io/badge/pantheon-unofficial-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/unofficial?q=org%3Apantheon-systems)',
-            'deprecated' => '[![Deprecated](https://img.shields.io/badge/pantheon-deprecated-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/unofficial?q=org%3Apantheon-systems)',
+            'pilot' => '[![Pilot](https://img.shields.io/badge/Pantheon-Pilot-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#pilot)',
```
Comment (greg-1-anderson) [inline src/Util/SupportLevel.php:40]: I guess we'll drop pilot again.

File: src/Cli/OrgCommands.php [hunk 2]
```diff
@@ -226,6 +228,8 @@ public function orgUpdateProjectsInfo($csv_file, $options = [
                             $this->logger->warning(sprintf("Skipping project %s because of empty or invalid support level.", $projectFullName));
                             continue;
                         }
+                    } elseif ($projectSupportLevel === 'Unsupported') {
```
Comment (greg-1-anderson) [inline src/Cli/OrgCommands.php:231]: If we don't combine unsupported and deprecated into a single support level, this conditional should probably also use the unsupported text for deprecated projects

**Review by greg-1-anderson — APPROVED** [review]

File: src/Cli/OrgCommands.php (hunk 2 above)
Comment (kporras07) [reply src/Cli/OrgCommands.php]: Done

File: src/Util/SupportLevel.php (hunk 1 above)
Comment (kporras07) [reply src/Util/SupportLevel.php]: Done

**Comment by kporras07** [comment]
Setting as Ready for Review/Merge and waiting for final confirmation in Slack before merging

**Pull request merged**